### PR TITLE
Add TopNav Component

### PR DESCRIPTION
**Pull request by eddier**
Adds the TopNav component.  See Styleguide for live example.  If there is feedback on code syntax we can discuss here.

```
      <TopNav>
        <TopNav.Content align="left">
            <img src={logo.png} alt="Alt text" />
        </TopNav.Content>
        <TopNav.Content align="right">
              Users Name
              <TopNav.LocalTime />
        </TopNav.Content>
      </TopNav>

```

### CODE REVIEW

**Comment by cdaringe** [comment]
surgereview23793815.surge.sh

File: src/components/TopNav/TopNav.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+  render () {
+    return (
+      <div className='nav__main clearfix'>
+        {this.props.children}
+      </div>
+    )
+  }
+}
+
```
Comment (cdaringe) [inline src/components/TopNav/TopNav.jsx]: can we do a `.propTypes = {}` so we get the pretty prop docs?

**Comment by wa11-e** [comment]
Woohoo! new surge deployment available for viewing! :tada: [surgereview23872288.surge.sh](http://surgereview23872288.surge.sh)

**Comment by wa11-e** [comment]
Woohoo! new surge deployment available for viewing! :tada: [surgereview23890158.surge.sh](http://surgereview23890158.surge.sh)

**Comment by eddier** [comment]
@cdaringe I tried to clean up the docs a bit for nested components without actually writing individual example files for each of them.  Can you check it out on this branch and see if you're ok with that?  I think the styleguidist team will make a proper fix in a future release given the thread I linked earlier.

**Comment by cdaringe** [comment]
Whoops, sorry for the delay!  Looks good

**Pull request merged**